### ISSUE TITLE
<refines> clarification

### ISSUE THREAD

**Issue by rickj** [w3c/publ-a11y#374]
There is one instance of refines in the techniques guide, as:
/package/metadata/meta[@property="dcterms:date" and @refines=//meta[@property="a11y:certifiedBy"]/@id]/text()

https://www.w3.org/TR/epub-a11y-11/  presents multiple examples of the use of refines for metadata other than as a date for certifiedBy.

Should:

1. only the instance in the techniques guide be presented thru the user interface?
2. only the examples in the EPUB 1.1 specification be presented?
3. Every metadata item potentially accommodate a refines to augment the display to the user?

**Comment by rickj** [comment]
The wording of https://www.w3.org/TR/epub-33/#attrdef-refines makes it sound like anything could have a refines attached to it.

**Comment by mattgarrish** [comment]
As I mentioned on the call yesterday, the only case for refines for accessibility metadata is on the certifier information you've noted in the Accessibility 1.1 specification.

Looking at the epub techniques, it doesn't account for all the possibilities. I don't know if it matters a lot, as it's probably unlikely that there will be more than one certifier for a publication, so any reports and credentials will belong to that entity. But that's kind of a lazy approach that could fail in weird ways.

You can find refined metadata elsewhere, as Hadrien mentioned, like for creators, contributors, titles, and subjects, but since those aren't in scope of this document I assume you'd be handling them elsewhere in your metadata display to users.

**Comment by clapierre** [comment]
For example, Macmillan Learning has two conformsTo statements one for an internal conformance accessibility standard and then the WCAG 2.1 AA statement and there should be in this situation two certifiedBy with one being Macmillan themselves and the 2nd being a 3rd party certification which in our case is by Benetech with our GCA credential.

**Comment by mattgarrish** [comment]
> Macmillan Learning has two conformsTo statements

Then it should be incorporated into the xpaths. It probably complicates the display a bit, though.

**Comment by HadrienGardeur** [comment]
> Macmillan Learning has two conformsTo statements one for an internal conformance accessibility standard […]

How do they declare conformance to these internal accessibility standards? With a string? With a URI?

I still believe that using strings for conformance (internal or not) is the wrong move and a list of well-known URIs that could be documented somewhere would work better.

URIs have the benefit of being dereferenceable, which could point to a page where Macmillan Learning would provide additional details about their own internal a11y standards. They're also much easier to validate and compare, which are important properties when used in the kind of `if` statements that we're documenting.

**Comment by clapierre** [comment]
> How do they declare conformance to these internal accessibility standards? With a string? With a URI?

With a string @HadrienGardeur 
    <meta property="dcterms:conformsTo">Macmillan Learning Flat Accessible ePub Specification V5.1</meta>

We will encourage Macmillan to add the following

    <meta id="macmilan-accessibility" property="dcterms:conformsTo">Macmillan Learning Flat Accessible ePub Specification V5.1</meta>
    <meta property="a11y:certifiedBy" refines="#macmilan-accessibility">Macmillan Learning</meta>

Similarly for the conformance to WCAG and 3rd party certification.

**Comment by clapierre** [comment]
This may still be a problem with the current 2.0 guidelines for the EPUB Techniques if there are more than one conformsTo and then who certifiedBy may be an issue if there are multiple certifiedBy using the refines to the conformstTo statement.
